tools/topology/topology2/cavs-mixin-mixout-hda.conf

```diff
@@ -28,7 +31,7 @@ Object.Dai.HDA [
 ]
 
 Object.Pipeline {
-	mixout-gain-dai-copier-playback [
+	mixout-gain-efx-dai-copier-playback [
```

Comment:
Sorry, I think it's the wrong way of proceeding. I talked to @ranj063 on this many months ago and what we need is a way to have nested pipelines, where additional elements are added to a baseline.

So the baseline is mixout-dai-copier, and we should be able to add gain, eq, drc, 3rd party with extensions rather than the creation of a new pipeline all the time.

Let's not add this by default, we will pay it dearly when we want to trim down the firmware for performance or early FPGA work with limited resources.